assets/app/view/game/corporation.rb

```diff
@@ -413,8 +413,7 @@ def render_shares
         if player_rows.any? || @corporation.num_market_shares.positive?
           at_limit = @game.share_pool.bank_at_limit?(@corporation)
           double_certs = @game.share_pool.shares_of(@corporation).count(&:double_cert)
-          step = @game.round.active_step
-          other_flags = step.respond_to?(:share_flags) && step&.share_flags(@game.share_pool.shares_of(@corporation))
+          other_flags = @game.respond_to?(:share_flags) && @game.share_flags(@game.share_pool.shares_of(@corporation))
```

Comment:
If you are going to add a @game method, you don't need to use respond_to?, instead add it to the Game::base class.

Reply:
Done